backend/generated/schema.graphql

```diff
@@ -29,7 +29,7 @@ type Query {
   userBalances(where: userBalanceFilter, orderBy: String, orderDirection: String, before: String, after: String, limit: Int): userBalancePage!
   relayBridge(chainId: Float!, contractAddress: String!): relayBridge
   relayBridges(where: relayBridgeFilter, orderBy: String, orderDirection: String, before: String, after: String, limit: Int): relayBridgePage!
-  bridgeTransaction(originChainId: Float!, originBridgeAddress: String!, nonce: BigInt!): bridgeTransaction
+  bridgeTransaction(originChainId: Float!, originBridgeAddress: String!, nonce: Float!): bridgeTransaction
```

Comment:
why is nonce a float here ?

Reply:
Also as this is a generated file, should we not check it inside the repo ?

Reply:
No we need to keep it in te repo as this is what generates the graphql client. Not sure why it sets it as a Float. I will investiagte!